nsv13/code/modules/food_and_drinks/drinks/nsv13_drinks.dm

```diff
@@ -0,0 +1,37 @@
+//Literally I made these so they can come up in your mail, but honestly do what you want, boo boo
+
+/obj/item/reagent_containers/food/drinks/drinkingglass/doctor_delight
```

Comment:
Rename this file to custom_drinks.dm as there is no need for nsv13 in the title as it is in the nsv13 folder.
Also we don't label files in that manner.